code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -61,9 +62,11 @@
 	name = "medical voidsuit"
 	desc = "A special suit that protects against hazardous, low pressure environments. Has minor radiation shielding."
 	item_state = "medical_voidsuit"
-	allowed = list(
-		/obj/item/device/lighting/toggleable/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,
-		/obj/item/weapon/storage/firstaid,/obj/item/device/scanner/healthanalyzer,/obj/item/stack/medical
+	extra_allowed = list(
+		/obj/item/weapon/storage/firstaid,
+		/obj/item/device/scanner/healthanalyzer,
+		/obj/item/stack/medical,
+		/obj/item/roller
```

Comment:
One can wear collapsable bed in suit_store?

Reply:
like the toolbox, this was already here. only for medical suits
Imagine it as being worn across the back